content/docs/guides/vote_poll_sdk/dapp_creation/03_connect_walle_useDapp.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Connect to the Ethereum Wallet
+title: Connect to the Ethereum Wallet useDapp
```

Comment:
`title: Connect to the Ethereum Wallet with useDapp`
The title shows in the side bar so best if it's a complete sentence.